etc/start_faux

```diff
@@ -52,8 +52,8 @@ ip route
 if [ -n "${options[dnsmasq]}" ]; then
     echo Configuring dnsmasq...
     echo dhcp-range=192.168.1.10,192.168.1.255 >> /etc/dnsmasq.conf
-    echo dhcp-host=9a:02:57:1e:8f:01,192.168.1.1 >> /etc/dnsmasq.conf
-    echo dhcp-host=9a:02:57:1e:8f:02,192.168.1.2 >> /etc/dnsmasq.conf
+    echo dhcp-host=$FAUX_MAC_PREFIX:8f:01,192.168.1.1 >> /etc/dnsmasq.conf
```

Comment:
Is it possible to use a command-line arge to start_faux?  There's already the mechanism for command-line options w/ args...